pkgs/development/python-modules/mlflow/default.nix

```diff
@@ -74,10 +75,17 @@ buildPythonPackage rec {
     "mlflow"
   ];
 
+  # Some mlflow subcommands like `mlflow server` run the gunicorn binary which
+  # in turn attempts to find the mlflow package again. Gunicorn does not have
+  # mlflow in its closure and won't find it unless we expose it here.
+  makeWrapperArgs = ''--prefix PYTHONPATH ":" "$PYTHONPATH"'';
```

Comment:
also not when you use `python3.withPackages`?

Reply:
> also not when you use `python3.withPackages`?

Yes, without this `makeWrapperArgs`, it results in `ModuleNotFoundError: No module named 'mlflow'`.